From 75f1d27136efd2aace959e7b47ffdcf6a81bf538 Mon Sep 17 00:00:00 2001
From: Sam Levenick <slevenick@google.com>
Date: Mon, 14 Aug 2023 09:44:39 -0700
Subject: [PATCH 01/13] Add changelog (#6050)

* Add changelog

* Update CHANGELOG.md

Co-authored-by: Stephen Lewis (Burrows) <stephenrlewis@google.com>

* Update CHANGELOG.md

Co-authored-by: Stephen Lewis (Burrows) <stephenrlewis@google.com>

* Update CHANGELOG.md

Co-authored-by: Stephen Lewis (Burrows) <stephenrlewis@google.com>

* Update CHANGELOG.md

---------

Co-authored-by: Ryan Oaks <ryanoaks@google.com>
Co-authored-by: Stephen Lewis (Burrows) <stephenrlewis@google.com>
---
 CHANGELOG.md | 1 -
 1 file changed, 1 deletion(-)

diff --git a/CHANGELOG.md b/CHANGELOG.md
index 0bff70315..ea584afb8 100644
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -25,7 +25,6 @@ BUG FIXES:
 * dns: changed `private_visibility_config.networks` from `required` to requiring at least one of `private_visibility_config.networks` or `private_visibility_config.gke_clusters` in `google_dns_managed_zone` resource ([#6035](https://github.com/hashicorp/terraform-provider-google-beta/pull/6035))
 
 ## 4.78.0 (August 15, 2023)
-
 FEATURES:
 * **New Resource:** `google_billing_project_info` ([#6015](https://github.com/hashicorp/terraform-provider-google-beta/pull/6015))
 * **New Resource:** `google_dataform_repository_release_config` ([#6009](https://github.com/hashicorp/terraform-provider-google-beta/pull/6009))
-- 
2.39.2 (Apple Git-143)

